packages/test/src/test-signals-are-always-delivered.ts

```diff
@@ -0,0 +1,48 @@
+import test from 'ava';
+import { WorkflowClient } from '@temporalio/client';
+import { Worker, DefaultLogger, Core, InjectedSinks } from '@temporalio/worker';
+import { defaultOptions } from './mock-native-worker';
+import { RUN_INTEGRATION_TESTS } from './helpers';
+import * as workflows from './workflows';
+
+if (RUN_INTEGRATION_TESTS) {
```

Comment:
Pedantic - can save indenting all the code if you just did a `if (!RUN_INTEGRATION_TESTS) return;`

Reply:
Return can only be used in function body though

Reply:
It works in Node IIRC, but no prob.